packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -107,6 +108,9 @@ class TestCommand extends FastFlutterCommand {
         'called *_test.dart and must reside in the package\'s \'test\' '
         'directory (or one of its subdirectories).');
     }
+
+    await _buildTestAsset();
```

Comment:
This needs to happen after the pub check, so that any assets from package dependencies will already exist.

Reply:
fixed